pkgs/development/interpreters/jruby/default.nix

```diff
@@ -1,27 +1,28 @@
-{ lib, stdenv, callPackage, fetchurl, makeWrapper, jre }:
+{ lib, stdenv, callPackage, fetchurl, makeBinaryWrapper, jre }:
 
 let
 # The version number here is whatever is reported by the RUBY_VERSION string
-rubyVersion = callPackage ../ruby/ruby-version.nix {} "2" "5" "7" "";
+rubyVersion = callPackage ../ruby/ruby-version.nix {} "3" "1" "4" "";
 jruby = stdenv.mkDerivation rec {
```

Comment:
Hi!

I added a non-blocking comment, feel free to implement it if you want, if not, just let me know and I'll merge this immediately.

Reply:
What's the comment ?

Reply:
Looks like it was lost, I don't know why. Let me put it back.

Reply:
Thanks for reviewing. I refactored the expression to use the `finalAttrs` pattern.